sys/include/crypto/modes/ccm.h

```diff
@@ -42,6 +42,10 @@ extern "C" {
  */
 #define CCM_BLOCK_SIZE                      16
 
+/**
+ * @brief Maximum length for the appended MAC
+ */
+#define CCM_MAC_MAX_LEN                     16
```

Comment:
Should this go upstream in a separate PR?

Reply:
This define was never needed? I think putting this line into an own commit should be sufficient.

Reply:
It was not needed before, but @kaspar030 had made a comment about avoiding VLA, and the max size for the MAC in CCM is 16 bytes, I preferred adding a define to the header than defining it in the openwsn code.

I could add it in its own PR, but it would have no use case, same for d924a3ff0d41344209287ae2401da9ec4bc56b5a